doc/manual/src/release-notes/rl-next.md

```diff
@@ -2,5 +2,16 @@
 
 - Fixed a bug where `nix-env --query` ignored `--drv-path` when `--json` was set.
 
-- Introduced the store [`mounted-ssh-ng://`](@docroot@/command-ref/new-cli/nix3-help-stores.md).
-  This store allows full access to a Nix store on a remote machine and additionally requires that the store be mounted in the local filesystem.
+- Introduce experimental mounted/unmounted variations to our remote stores:
+
+  - [`mounted-ssh-ng://`](@docroot@/command-ref/new-cli/nix3-help-stores.md)
+
+    This store allows full access to a Nix store on a remote machine and additionally requires that the store be mounted in the local filesystem.
+
+  - [`unmounted-unix://`](@docroot@/command-ref/new-cli/nix3-help-stores.md)
```

Comment:
Getting a sense of impending combinatorial explosion here.
Mounted or not seems orthogonal to the protocol used.
Inheritance isn't suitable for arbitrary composition of orthogonal behaviors, so this suggests that the Store hierarchy should be refactored instead, to use value composition instead of inheritance for this purpose.
As far as URIs, that could exhibit as a more regular format such as

```
ssh-ng://foo
ssh-ng://foo?mounted=true
unix://?mounted=false
unix://
```

The only remaining irregularity is in the defaults, but I feel like that's expected and part of the domain.

Reply:
*Perhaps* the mount could be considered part of the protocol, and in a way it is, but then it should be part of the scheme, where we don't have a syntax for removing a protocol. E.g. `ssh-ng+mount:` would be ok, but `unix-mount:` makes no sense out of context.

Main point is the explosion of subclasses, not the syntax.

Reply:
Well-documented, explicitly experimental stop gap is ok.